src/main/java/duke/exception/InvalidIndexException.java

```diff
@@ -0,0 +1,10 @@
+package duke.exception;
+
+/**
+ * An Exception class that encapsulates the situation where the user has given an invalid index to a task.
+ */
```

Comment:
I like how meticulous you are with your JavaDocs, writing clear documentation for all of your Exception classes 👍